x-pack/test/api_integration/apis/cloud_security_posture/benchmark.ts

```diff
@@ -119,7 +129,7 @@ export default function ({ getService }: FtrProviderContext) {
         .set('kbn-xsrf', 'xxxx')
         .expect(200);
 
-      expect(res.items.length).equal(1);
+      expect(res.items.length).equal(2);
```

Comment:
We are creating a vulnerability Integration so test cases are factoring vuln_mgmt integrations being filtered out.
```
await createPackagePolicy(
        supertest,
        agentPolicyId3,
        'vuln_mgmt',
        'cloudbeat/vuln_mgmt_aws',
        'aws',
        'vuln_mgmt',
        'CNVM-1'
      );```

Reply:
yes, this is true. What I meant though was that it's a good practice for tests to test specific cases. Eg. from the test `Should return array size 2 when we set per page to be only 2 (total element is still 3)` it's hard to say what logic exactly we are testing. It would be nice to have a specific test "should filter out vulnerability policy" so that this specific logic is tested explicitly, not as a side effect of another test. But it's not a big deal ofc.

Reply:
btw we also have two unit tests with the same name `Should return array size 2 when we set per page to be only 2 (total element is still 3)`, the second test is just testing the second page of the pagination (and it returns the array size 1). anyway after some getting back to these tests will be very confusing